benchmark/net/chunky_http_client.js

```diff
@@ -0,0 +1,111 @@
+// test HTTP throughput in fragmented header case
+var common = require('../common.js');
+var server;
+var test = require('../../test/common.js');
+var PORT = common.PORT;
+var PIPE = test.PIPE;
+
+var net = require('net');
+var bench = common.createBenchmark(main, {
+  len: [1, 4, 8, 16, 32, 64, 128],
+  num: [5, 50, 500, 2000],
+  type: ['send'],
+}
+)
+
+var len;
+var num;
+var type;
```

Comment:
I would suggest moving the vardecls into the function that uses them.  JS being what it is, this makes it too easy to accidentally leak variables.  Also, having a `'use strict'` at the top is never a bad thing.